ultimate-kata-vue/src/components/question.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div :class="[mainClass, background]">
```

Comment:
You may not need to add css classes in variables like this unless the css class name in dynamic.

```
if (button === pressed)
 return 'bg-green 
else 
 return 'bg-red'

Reply:
can just be `class="question code-bg"`

Reply:
```suggestion
  <div class="question code-bg">
```
I shall push this next